src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,512 @@
+import React from 'react'
+
+import { CardGroup } from './CardGroup/CardGroup'
+import { Card } from './Card/Card'
+import { CardHeader } from './CardHeader/CardHeader'
+import { CardFooter } from './CardFooter/CardFooter'
+import { CardBody } from './CardBody/CardBody'
+import { CardMedia } from './CardMedia/CardMedia'
+import { Button } from '../Button/Button'
+
+export default {
+  title: 'Card',
+  parameters: {
+    info: `
+USWDS 2.0 Card components
+
+Source: https://designsystem.digital.gov/components/card/
+`,
+  },
+}
+
+const card = (
+  <Card gridLayout={{ tablet: { col: 6 } }}>
```

Comment:
can we have all of our cards here be `col: 4` so its easier to see them side by side.  Now that we have the test stories also included there still are opportunities to see cards with different grid classes applied elsewhere

Reply:
sure